runtime/onert/core/src/ir/OperationValidator.cc

```diff
@@ -211,6 +211,14 @@ void OperationValidator::visit(const operation::DepthToSpace &node)
   OP_REQUIRES(block_size > 0);
 }
 
+void OperationValidator::visit(const operation::DetectionPostProcess &node)
+{
+  auto param = node.param();
+
+  // FIXME: number of classes should be 1 for now.
+  OP_REQUIRES(param.num_classes == 1);
```

Comment:
If this kernel support `num_classes > 1`, we can test kernel by using tflite's unittest value (`tensorflow/lite/kernels/detection_postprocess_test.cc`)...